lib/ddtrace/workers.rb

```diff
@@ -38,9 +66,17 @@ def start
         @run = true
```

Comment:
It's an easy check here.... but can we add a guard condition like:
```ruby
return unless !@run
```
so that this action doesn't start new workers if called twice? This prevents thread pollution and memory leaks if used improperly.